packages/pigment-css-react/tests/useTheme/fixtures/useTheme.output.js

```diff
@@ -0,0 +1,117 @@
+import { useTheme as _useTheme } from '@pigment-css/react';
```

Comment:
Where is the runtime `useTheme`?

Reply:
Here's the main PR now - https://github.com/mui/pigment-css/pull/105

See the description.